resources/views/home.blade.php

```diff
@@ -1,22 +1,113 @@
+                            alt="{{ $category->name }}">
+                    </div>
+                    <div class="category">
+                        <div class="display-table center-text">
+                            <div class="display-table-cell">
+                                <h3><b>{{ $category->name }}</b></h3>
+                            </div>
                         </div>
-                    @endif
-
-                    {{ trans('logged_in') }}
-                </div>
-            </div>
-        </div>
-    </div>
-</div>
+                    </div>
+                </a>
+            </div><!-- swiper-slide -->
+            @empty
+
+            <div class="swiper-slide">
+                <strong>{{ trans('no_data') }}</strong>
+            </div><!-- swiper-slide -->
+            @endforelse
+
+        </div><!-- swiper-wrapper -->
+    </div><!-- swiper-container -->
+</div><!-- slider -->
+<section class="blog-area section">
+    <div class="container">
+        <div class="row">
+            @forelse ($posts as $post)
+            <div class="col-lg-4 col-md-6">
+                <div class="card h-100">
+                    <div class="single-post post-style-1">
+                        <div class="blog-image"><img
+                                src="{{ asset('assets/frontend/images/averie-woodard-319832.jpg') }}"></div>
+                        <a class="avatar" href="#">
+                            <img src="{{ asset('bower/adminbsb-materialdesign/images/image-gallery/15.jpg') }}"
+                                alt="Profile Image">
+                        </a>
+                        <div class="blog-info">
+                            <h4 class="title"><a href="{{ route('post', $post->id) }}"><b>{{ $post->title }}</b></a>
+                            </h4>
+                            @guest
+                            <ul class="post-footer">
+                                <li>
+                                    <a href="{{ route('login') }}"><i class="ion-heart"></i>
+                                        {{ $post->favorite_to_users()->count() }}
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="{{ route('login') }}"><i class="ion-chatbubble"></i>3</a>
+                                </li>
+                                <li>
+                                    <a href="{{ route('login') }}"><i class="ion-eye"></i>{{ $post->view_count }}</a>
+                                </li>
+                            </ul>
+                            @else
+                            <ul class="post-footer">
+                                <li>
+                                    <a href="{{ route('post', $post->id) }}"><i class="ion-heart"></i>
+                                        {{ $post->favorite_to_users()->count() }}
```

Comment:
```suggestion
                                        {{ $post->favorite_to_users->count() }}
```
nếu có dấu ngoặc thì sẽ là query, không có thì mình sẽ count từ collection

Reply:
ok t sẽ sửa lại